src/components/InjectedComponents/PostDialog.tsx

```diff
@@ -227,7 +227,7 @@ export function PostDialogUI(props: PostDialogUIProps) {
                             </SelectRecipientsUI>
                         </Box>
                         {/* This feature is not ready for mobile version */}
-                        {webpackEnv.genericTarget === 'browser' ? (
+                        {webpackEnv.genericTarget !== 'facebookApp' ? (
```

Comment:
imgPayload on chrome working for both twitter/fb right?

Reply:
yes！